examples/receive.rs

```diff
@@ -1,18 +1,12 @@
 use anyhow::Result;
 use linux_embedded_hal::spidev::{SpiModeFlags, SpidevOptions};
-use linux_embedded_hal::sysfs_gpio::Direction;
-use linux_embedded_hal::{Spidev, SysfsPin};
+use linux_embedded_hal::{SpidevDevice, SysfsPin};
 use rfm69::Rfm69;
 use utilities::rfm_error;
 
 fn main() -> Result<()> {
-    // Configure CS pin
-    let cs = SysfsPin::new(25);
```

Comment:
There is missed `SysfsPin` usage at line 40 (34 after change).

Reply:
The idea was to demonstrate how to use `SpidevDevice` in this example, where the device driver takes care of managing the associated CS pin automatically.

Reply:
Right that's fine, what I meant is that this bit is not needed then `SysfsPin::new(25).unexport()?;`

Reply:
Ah right sorry, I misread your comment. Fixed.